packages/next/client/image.tsx

```diff
@@ -59,14 +60,15 @@ export type ImageProps = Omit<
       }
   )
 
+const config =
+  ((process.env.__NEXT_IMAGE_OPTS as any) as ImageConfig) || imageConfigDefault
+const configLoader = config.loader || 'default'
```

Comment:
This default is already handled when reading `next.config.js` if I remember correctly

Reply:
Oops, yep that was an artifact from the previous version. Fixed!